package.json

```diff
@@ -24,12 +24,13 @@
     "test:watch": "npm run test:coverage --watch",
     "test": "npm run test:coverage",
     "pretest": "npm run lint",
-    "prepare": "rimraf ./ssl/*.pem && npm run -s transpile:index && npm run transpile:clients && npm run -s build:live && npm run -s build:index && npm run -s build:sockjs",
+    "prepare": "rimraf ./ssl/*.pem && npm run build:client",
     "transpile:index": "babel client-src/default --out-dir client --ignore \"./client-src/default/*.config.js\"",
     "transpile:clients": "babel client-src/clients --out-dir client/clients",
-    "build:index": "webpack ./client-src/default/index.js -o client/index.bundle.js --color --config client-src/default/webpack.config.js",
-    "build:live": "webpack ./client-src/live/index.js -o client/live.bundle.js --color --config client-src/live/webpack.config.js",
-    "build:sockjs": "webpack ./client-src/sockjs/index.js -o client/sockjs.bundle.js --color --config client-src/sockjs/webpack.config.js",
+    "build:client:index": "webpack ./client-src/default/index.js -o client/index.bundle.js --color --config client-src/default/webpack.config.js",
+    "build:client:live": "webpack ./client-src/live/index.js -o client/live.bundle.js --color --config client-src/live/webpack.config.js",
+    "build:client:sockjs": "webpack ./client-src/sockjs/index.js -o client/sockjs.bundle.js --color --config client-src/sockjs/webpack.config.js",
+    "build:client": "rimraf ./client/* && npm-run-all -s -l -p \"build:client:**\"",
```

Comment:
- We don't extract `rm` scripts because it doesn't have sense, nobody will be used their separate like `npm run rm:ssl` 
- We use `build:client` because we should migrate on `babel` in next major release for server, so we have these command in `prepare`:
```
npm run build:client - build client
npm run build:server - build server (in next major release when we migrate on babel)
npm run build - build client and server
```

Reply:
Why do you use babel at the server?

Reply:
Using all the features ES like async/await (already supported in node@8), private fields, `import`/`export` and etc.

Reply:
It is not high priority but will be great to have consistent ecma 2018/2019 code in client and server

Reply:
ok, I want to use typescript.😉 I need types..lol

Reply:
I thought Node 6 was being dropped so we could use async/await, etc. without problems?

Reply:
ESM is not supported.

Reply:
Yes ESM is not supported, it doesn't matter if we only use babel or typescript we will build server so i rename this command, approve?

Reply:
Yes, approved